socorro/external/es/correlations.py

```diff
@@ -96,9 +92,21 @@ def _prefix_to_datetime_date(self, prefix):
         return datetime.date(yy, mm, dd)
 
     @staticmethod
-    def make_id(document):
+    def make_id(document, only_keys=None):
+        only_keys = only_keys or (
+            'platform',
+            'product',
+            'version',
+            'date',
+            'key',
+            'signature',
+            'count',
```

Comment:
I think ``count`` should not be in there. It could change when you reload the job, if new crashes came in in the mid-time for example. I can happen if someone triggers the processing of a crash that has been throttled. Or say we had a problem with processing, and we want to re-run correlations a bit later when all crashes made it into the database.